daemon/cmd/daemon.go

```diff
@@ -558,11 +569,16 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup, params *daemonParams
 				oldNIDs = append(oldNIDs, nid)
 			} else if nid == identity.ReservedIdentityIngress && v.TunnelEndpoint.IsZero() {
 				oldIngressIPs = append(oldIngressIPs, k.IPNet())
-				ip := k.IPNet().IP
-				if ip.To4() != nil {
-					node.SetIngressIPv4(ip)
-				} else {
-					node.SetIngressIPv6(ip)
+				if addr, ok := netip.AddrFromSlice(k.IPNet().IP); ok {
```

Comment:
Please don't use `AddrFromSlice` to convert net.IP to netip.Addr. Use ip.AddrFromIP instead.

Reply:
We can see several other places using `AddrFromSlice`. Please fix them as well.